src/inspectors/ensureIssuerSignature.js

```diff
@@ -0,0 +1,25 @@
+import { BLOCKCHAINS } from '../constants';
+import bitcoin from 'bitcoinjs-lib';
+import VerifierError from '../models/verifierError';
+
+export default function ensureIssuerSignature (
+  issuerKey,
+  certificateUid,
+  certificateSignature,
+  chain
+) {
+  let bitcoinChain =
```

Comment:
Interesting, does ethereum not do this call?

Reply:
@AnthonyRonning not sure... I kept what was in the v1 here: https://github.com/blockchain-certificates/cert-verifier-js/blob/master/lib/checks.js#L54

Reply:
Ohh okay since that's a v1 specific call and we didn't have ethereum in v1, then that's okay! I was worried I missed a step when adding ethereum back then.